commands/initial/templates/package.json

```diff
@@ -15,7 +15,9 @@
     "prebuild": "rimraf dist out-tsc",
     "start": "webpack-dev-server --open --config ./webpack/webpack.dev.js",
     "tagVersion": "np --no-publish",
-    "test": "node ./tasks/test"
+    "test": "node ./tasks/test",
+    "pree2e": "webdriver-manager update",
+    "e2e": "protractor"
```

Comment:
Oh, sorry, I didn't mean along with unit tests. I do understand that protractor testes can be time-consuming.

What I meant was would we be able to leverage the current `tasks/test.js` file to execute the e2e tests? That way we keep the tests running all in one file.